openfl/plugins/frameworks_adapters/keras_adapter.py

```diff
@@ -45,7 +47,9 @@ def __reduce__(self):  # NOQA:N807
             cls.__reduce__ = __reduce__
 
         # Run the function
-        make_keras_picklable()
+        tf_version = float((tf.__version__).split(".")[0] + "." + (tf.__version__).split(".")[1])
```

Comment:
`tf.__version__` supports direct string boolean checks. We could simplify this to

```python
if tf.__version__ < '2.7.1':
  ...
```

Reply:
@mansishr A few minor corrections on the MNIST experiment. Will push the commits here before this gets merged. Is that OK?